src/utils/statusColumnUtils.ts

```diff
@@ -46,43 +47,36 @@ function mapAktivitetskravStatus(personData: PersonData): string {
     case AktivitetskravStatus.FORHANDSVARSEL:
       const svarfrist =
         personData.aktivitetskravvurdering?.vurderinger[0]?.varsel?.svarfrist;
-      const fristDate = svarfrist ? new Date(svarfrist) : undefined;
-      const today = new Date();
 
-      if (fristDate && fristDate >= today) {
-        return '- Forhåndsvarsel sendt';
-      } else {
-        return '- Forhåndsvarsel utløpt';
+      if (svarfrist) {
+        return isPast(svarfrist)
+          ? '- Forhåndsvarsel utløpt'
+          : '- Forhåndsvarsel sendt';
       }
+
+      return '';
     default:
       return '';
```

Comment:
Hvis man ikke har `return` eller `break` i switch-casen, vil den fortsette til `default` da?

Reply:
Godt spørsmål, kan lage en test og sjekke :sweat_smile:

Reply:
Får `Fallthrough in 'switch' statement. Missing 'break' or 'return'` hvis denne fjernes, så da tenker jeg `return '';` er bedre og mer eksplisitt enn `break` her